roles/openshift_excluder/tasks/disable.yml

```diff
@@ -1,47 +1,38 @@
 ---
-# input variables
-# - excluder_package_state
-# - docker_excluder_package_state
-- include: init.yml
+- when: r_openshift_excluder_verify_upgrade
```

Comment:
Yeah, that's a really interesting way of getting the "if <condition>" feel in ansible.

+1